contracts/IRelayRecipient.sol

```diff
@@ -29,10 +29,22 @@ contract IRelayRecipient {
      *          the contract may check the method-id for valid methods
      *  @param gasPrice - the gas price for this transaction
      *  @param transactionFee - the relay compensation (in %) for this transaction
-     *  @param signature - sender's signature over all parameters except approvalData
      *  @param approvalData - extra dapp-specific data (e.g. signature from trusted party)
      */
-    function acceptRelayedCall(address relay, address from, bytes memory encodedFunction, uint gasPrice, uint transactionFee, bytes memory signature, bytes memory approvalData) public view returns (uint);
+    function acceptRelayedCall(
+        address relay,
+        address from,
+        bytes memory encodedFunction,
+        uint256 transactionFee,
+        uint256 gasPrice,
+        uint256 gasLimit,
+        uint256 nonce,
+        bytes memory approvalData,
+        uint256 maxPossibleCharge
```

Comment:
Simply because the cost of passing it along is very low, whereas calculating it (again, recall that RelayHub needs to compute this value anyway) can get expensive.